### PR TITLE
Adding link for detailed instructions

### PR DESCRIPTION
**Pull request by keblek**
LiT links to this after installing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ Lightning Terminal (LiT) is a browser-based interface for managing channel liqui
 - Use Pool to earn sats by opening channels to those needing inbound liquidity
 
 ## Installation
-Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page.
+Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page. 
+
+Additionally you find detailed instructions in the [lightning engineering docs](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/get-lit).
```
Comment (guggero) [inline README.md:22]:
```suggestion
Additionally you find detailed instructions on the [docs.lightning.engineering](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/get-lit) page.
```

**Review by guggero — COMMENTED** [review]
Thanks for the PR, that link is definitely useful to have in there.
Just a small suggestion in the wording, otherwise LGTM!

**Comment by keblek** [comment]
Sure thing!

**Review by guggero — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ Lightning Terminal (LiT) is a browser-based interface for managing channel liqui
 - Use Pool to earn sats by opening channels to those needing inbound liquidity
 
 ## Installation
-Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page.
+Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page. 
+
+Additionally you find detailed instructions on the [docs.lightning.engineering](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/get-lit) page.
```
Comment (jamaljsr) [inline README.md:22]:
```suggestion
Additionally, you can find detailed instructions on the [docs.lightning.engineering](https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/get-lit) page.
```

**Review by jamaljsr — CHANGES_REQUESTED** [review]
Good idea to add this link. Thanks.

I just suggested a small grammar improvement.

**Review by matthiasdebernardini — APPROVED** [review]
thanks!

**Pull request merged**